project_app/pages/About.tsx

```diff
@@ -1,9 +1,60 @@
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import HomePage from '../components/HomePage';
+import {ProductContainer} from "../components/styled";
+import AboutWrapper from "../components/AboutWrapper";
+import{NavbarText} from "./style";
+import styled from "styled-components";
 
 
-function about() {
-  return (
-    <div>About</div>
-  )
-}
 
-export default about
\ No newline at end of file
+ function About() {
```

Comment:
pls change filename to about.tsx

Reply:
All filenames are lowercase 
<img width="253" alt="Screen Shot 2023-05-23 at 4 57 52 PM" src="https://github.com/idiglove/oku-no-kumo-fe/assets/34128735/f693c172-2b84-4c2d-bd9f-8f378f2d2b02">